cli/src/migrate.js

```diff
@@ -215,24 +231,48 @@ function makeBackup() {
     return Promise.resolve();
   }
 
-  return new Promise((resolve, reject) => {
-    white('Backing up rethinkdb_data directory');
-    const proc = child_process.spawn('rethinkdb', [
+  if (this.options.unportable_backup) {
+    return unportableBackup()
+  }
+
+  white('Backing up rethinkdb_data directory');
+  return procPromise('rethinkdb', [
       'dump',
       '--connect',
       `${rdbHost}:${rdbPort}`,
-    ]);
-    proc.on('exit', (code) => {
-      if (code === 0) {
-        green(' └── Backup completed');
-        resolve();
+    ]).then((proc) => {
+      green(' └── Backup completed');
+    }).catch((e) => {
+      if (e.message.match(/Python driver/)) {
+        throw new Error(`\
+The RethinkDB Python driver is not installed, so we can't \
+do a backup before migrating.
+
+You can install the Python driver with the instructions \
+found at:
+http://www.rethinkdb.com/docs/install-drivers/python/
+
+Alternately, you may pass the --unportable-backup flag \
+to hz migrate. This flag uses the tar command to make a \
+backup, but the backup is not safe to use on another \
+machine or to create replicas from.
```

Comment:
We should probably mention that:
1) this only works if the `rethinkdb_data` directory is local - that is, they normally run with `--start-rethinkdb`
2) this should not be performed while the `rethinkdb` server is running - the backup could have an inconsistent btree